madhav/src/main/java/com/lftechnology/phpjava/numbers/Fibonacci.java

```diff
@@ -0,0 +1,26 @@
+package main.java.com.lftechnology.phpjava.numbers;
+
+import main.java.com.lftechnology.phpjava.main.Scanner;
+
+public class Fibonacci {
+    
+    /**
+     * Displays sum of all even fibonacci numbers within given range
+     */
+    public void getFibonacciSum() {
+        System.out.println("Enter the limit:");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)